js/iframe-coordinator/package.json

```diff
@@ -22,19 +22,26 @@
     "karma-chrome-launcher": "^2.2.0",
     "karma-jasmine": "^1.1.2",
     "karma-webpack": "4.0.0-rc.2",
+    "prettier": "^1.14.3",
     "puppeteer": "^1.9.0",
     "source-map-loader": "^0.2.4",
     "ts-loader": "^5.0.0",
+    "tslint": "^5.11.0",
+    "tslint-config-prettier": "^1.15.0",
     "typescript": "^3.0.3",
     "webpack": "^4.16.3",
     "webpack-cli": "^3.1.0"
   },
   "scripts": {
     "test": "./node_modules/karma/bin/karma start --single-run",
-    "test-watch": "./node_modules/karma/bin/karma start",
-    "test-watch-chrome": "./node_modules/karma/bin/karma start --browsers=Chrome",
+    "test.watch": "./node_modules/karma/bin/karma start",
+    "test.watch.chrome": "./node_modules/karma/bin/karma start --browsers=Chrome",
```

Comment:
updated this format (using . for 'options') to match the commands used in web components.